### DISCUSSION TITLE
Notification Area Icons not working properly

### DISCUSSION THREAD

**Discussion by Guerrilla-RadiO** [valinet/ExplorerPatcher#3467] [General]
Hi, 

When I try to select some apps to show in the notification area and then apply changes, it won't work, the only possible option is to show all the icons which is annoying, I only can select either all the icons, or only the system ones, I can't select the icons that really matter for me. Below you can find my Windows version.

![image](https://github.com/valinet/ExplorerPatcher/assets/175247412/9fde1bf1-fdd4-480f-b1a9-0ccd84bbc03f)

**Comment by Guerrilla-RadiO** [comment]
I found the solution, just drag and drop the icons, please close this.

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1454

Please close this.
